test/integration/app-proxy-manager-agent.yaml.tmpl

```diff
@@ -40,7 +40,7 @@ spec:
             "image": "{{.hub}}/init:{{.tag}}",
             "args": ["-p", "15001", "-u", "1337"],
             "imagePullPolicy": "Always",
-            "securityContext": { "capabilities" : { "add" : ["NET_ADMIN"] } }
+            "securityContext": { "capabilities" : { "add" : ["NET_ADMIN"] }, "privileged": true }
```

Comment:
It might not be the case that the user who is able to reproduce the bug is capable of debugging the problem. There might also be bugs that are hard to reproduce again with debug image to capture the core dump.